packages/gatsby-remark-autolink-headers/README.md

```diff
@@ -58,6 +58,7 @@ Note: if you are using `gatsby-remark-prismjs`, make sure that it’s listed aft
 - `maintainCase`: Boolean. Maintains the case for markdown header (optional)
 - `removeAccents`: Boolean. Remove accents from generated headings IDs (optional)
 - `enableCustomId`: Boolean. Enable custom header IDs with `{#id}` (optional)
+- `isIconAfterH`: Boolean. Enable the anchor icon to be inline at the end of the header text
```

Comment:
i guess this is an optional option?

```suggestion
- `isIconAfterH`: Boolean. Enable the anchor icon to be inline at the end of the header text  (optional)
```

Reply:
Great idea! Made changes here https://github.com/gatsbyjs/gatsby/pull/19937/commits/81ade3671ef540666c4d6b6e0b29cb273d046964

Reply:
Looks like this is still not addressed? I mean appending ` (optional)` to this option description?